_rules/block-collapsible-3e12e1.md

```diff
@@ -0,0 +1,465 @@
+---
+id: 3e12e1
+name: Block of content is expandable and collapsible
+rule_type: atomic
+description: |
+  This rule checks that repeated blocks of content are expandable and collapsible
+accessibility_requirements:
+  wcag-technique:SCR28: # Using an expandable and collapsible menu to bypass block of content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS styling
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation 1
+
+There exists a [semantic segmentation][] of the test target such that for each [block][] in this [segmentation] which is before the [main block of content][] and contains at least one [block of repeated content][], there exists a [visible][] [instrument][] to make this [block][] not [visible][].
```

Comment:
I'm naming the block to make it clear.

> Also, cannot the block of repeated content be the other block? Does it have to be contained in the other block?

Yes, it can be the same. A set is included in itself 😉 I don't know if I should make that explicit. The examples are illustrating that since the blocks of the segmentation are exactly the blocks of repeated content.

Reply:
That is true. Perhaps the examples are enough.